app/src/main/java/com/dropbox/android/sample/Graph.kt

```diff
@@ -86,25 +92,25 @@ object Graph {
             })
         val adapter = moshi.adapter<RedditConfig>(RedditConfig::class.java)
         return StoreBuilder
-            .fromNonFlow<Unit, RedditConfig> {
+            .fromNonFlow<Unit, RedditConfig, RedditConfig> (fetcher = {
```

Comment:
this was something we tried to avoid in the new store builder. is it required now?